source/php/Component/Acceptance/acceptance.json

```diff
@@ -0,0 +1,33 @@
+{
+    "slug": "acceptance",
+    "default": {
+        "labels": false,
+        "modifier": "",
```

Comment:
May be replaced with boolean isVideo.

Reply:
Or type = "iframe"
Or type = "video"

?

Reply:
could change "modifier" to "type". Main reason is to make the usage area wider. This way you can easily style it for different cases not adding much code.